results/customer-portal/installer-platform.detailed-description.html

```diff
@@ -0,0 +1,12 @@
+The Red Hat JBoss Developer Studio Platform Installer scans for the products listed below using your PATH environment variable to determine if they're already installed. If not found, or an older version is installed, the following supported versions will then be installed for you by the installer:
+
+<ul>
+	<li> Vagrant 1.7.4
+	<li> VirtualBox 5.0.8
+	<li> OpenJDK 8.0
+	<li> Red Hat JBoss Developer Studio 9.1 GA
+	<li> Cygwin 2.5
+	<li> PuTTY 0.67
+	<li> OpenShift Client (OC) Tools 1.1.6
```

Comment:
all these version info is not relevant to put here imo. refer to the release notes/docs if you want all those details.